### ISSUE TITLE
GoToWindow.0.7.2.zip - I get this error

### ISSUE THREAD

**Issue by Stamimail** [christianrondeau/GoToWindow#53]
Startup Error
No plug-ins found. Check that at least GoToWindow.Plugins.Core.dll can be found in the Plugins directory and restart GoToWindow.

**Comment by christianrondeau** [comment]
Mmh I'll check this when I get a chance, in the meantime can you check in log4net.config, set some path for the log file and check if there's any error or warning in the file?

Also just in case, try disabling your anti-virus, and ensure the referenced dll is indeed there.

**Comment by Stamimail** [comment]
INFO  GoToWindow - Updates are disabled because GoToWindow is not running from the AppData directory. 
WARN  GoToWindow - Could not start listening to WMI events
System.Management.ManagementException: Access denied 
   ב-  System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   ב-  System.Management.ManagementEventWatcher.Start()
   ב-  GoToWindow.Api.WmiProcessWatcher.Start()
ERROR GoToWindow - System.Management.Instrumentation.InstanceNotFoundException: No plug-ins found
   ב-  GoToWindow.GoToWindowPluginsContainer.LoadPlugins()
INFO  GoToWindow - Application started. Shortcut is 'Left Win + Tab + Tab' (5B+09:2)
INFO  GoToWindow - Application exited.

**Comment by christianrondeau** [comment]
Mmmh, that's weird. I just uninstalled it, downloaded the zip file from GitHub directly, extracted it and everything works fine.

This error means the MEF engine (the thing that actually loads the behavior of GoToWindow from the executable) could not find any DLL file. The WMI events line is normal if you don't run GoToWindow as an administrator.

So we have to go to the basics;

1. Are you extracting the zip file in a folder, then running the executable? Or are you just opening the .zip folder and double-clicking on the .exe file?
2. Are you double-clicking on the executable, or running it from a command line?
3. Is there a `Plugins` subfolder in the extracted folder? This `Plugins` folder should contain a few DLL files, including `GoToWindow.Plugins.Core.dll`.

Did it work before? You might also want to try changing the log level to DEBUG and see if there's something helpful there.

**Comment by Stamimail** [comment]
1. extracting the zip file in a folder, then running the executable
2.  double-clicking on the executable
3. There is Plugins folder. There is file called GoToWindow.Plugins.Core.dll.
It just say it doesn't find it.

**Comment by christianrondeau** [comment]
Mmmh that's very weird. You did try without an anti-virus? Can you run from source? Otherwise I'll have to think of how to gather more clues...

**Comment by Stamimail** [comment]
> You did try without an anti-virus? 

No. No intention to disable the anti-virus, sorry.
I can only say that the anti-virus said nothing about those files while scanning.

**Comment by bybor** [comment]
For me, BitDefender [Enterprise] didn't let GoToMeeting to work. I had to ping them to tell it was false positive, and still got it blocked few times after getting AV updates. I never got deep enough to check the logs etc as you did, cannot confirm if that was similar to your errors.

**Comment by christianrondeau** [comment]
Yeah to be honest the AV is a shot in the dark, this specific error should be fairly simple.

I think adding some logs might help there. Can you build and run the app from visual studio? That will make it easier to diagnose the problem. 

Otherwise, maybe you are extracting in a unc path? E. G. \\somewhere? Or in Program Files directly?

**Comment by Stamimail** [comment]
D:\GoToWindow\GoToWindow.exe
Unicode can be the problem?

**Comment by christianrondeau** [comment]
Nope, should not make a difference here... Let me add some diagnostic logs tonight and see if that helps....

**Comment by christianrondeau** [comment]
Hi @Stamimail - try https://github.com/christianrondeau/GoToWindow/releases/tag/v0.7.3, and change this in the log4net.config file:

    <category name="GoToWindow">
      <level value="INFO" />
    </category>

Change `INFO` to `DEBUG`.

Then, run GoToWindow, and check the log file again.

Also make sure you extract in a clean folder to make sure we don's have conflicting versions.

**Comment by Stamimail** [comment]
2017-11-13 23:28:14 INFO  GoToWindow - Updates are disabled because GoToWindow is not running from the AppData directory. Executable: D:\GoToWindow.0.7.3\GoToWindow.exe, App Data: C:\Users\Stamimail\AppData\Local
2017-11-13 23:28:18 WARN  GoToWindow - Could not start listening to WMI events
System.Management.ManagementException: Access denied 
   ב-  System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   ב-  System.Management.ManagementEventWatcher.Start()
   ב-  GoToWindow.Api.WmiProcessWatcher.Start()
2017-11-13 23:28:20 DEBUG GoToWindow - Loaded directory catalog from 'D:\GOTOWINDOW.0.7.3\PLUGINS', looking for '*.dll'. Loaded: D:\GOTOWINDOW.0.7.3\PLUGINS\GOTOWINDOW.PLUGINS.CORE.DLL, D:\GOTOWINDOW.0.7.3\PLUGINS\GOTOWINDOW.PLUGINS.EXPANDBROWSERSTABS.DLL, D:\GOTOWINDOW.0.7.3\PLUGINS\GOTOWINDOW.PLUGINS.EXPLOREREXTENSIONS.DLL, D:\GOTOWINDOW.0.7.3\PLUGINS\LOG4NET.DLL
2017-11-13 23:28:20 ERROR GoToWindow - System.Exception: No plugins were composed
2017-11-13 23:28:20 DEBUG GoToWindow - Application activated.
2017-11-13 23:28:23 INFO  GoToWindow - Application started. Shortcut is 'Left Win + Tab + Tab' (5B+09:2)
2017-11-13 23:28:23 INFO  GoToWindow - Application exited.

**Comment by christianrondeau** [comment]
Well I'm stumped. The next step would be to go all in and do something like https://blogs.msdn.microsoft.com/dsplaisted/2010/07/13/how-to-debug-and-diagnose-mef-failures/ but then you'll need to build a custom mefx executable, etc. etc. Are you willing to go to the next step?

Also, do you have a C# environment to build the solution? The only variables I can think of are corrupted .NET installation, anti-virus preventing from loading the DLL files, bogus extract...
